### ISSUE TITLE
Rework "Source" display, possibly with glyphs

### ISSUE THREAD

**Issue by mvelinder** [iobio/clin.iobio#367]
![image](https://user-images.githubusercontent.com/22081541/109985896-74cca380-7cc2-11eb-9f29-e965c5c4d41a.png)

Source: has never been intuitive and needs a rework. Numbers aren't good for this in my opinion. I would suggest small glyphs like the ClinVar and Mode of Inheritance glyphs we are already using. 

The current and proposed glyphs would then be:
(1) = "Computationally prioritized" and use a computer or server stack or some other "computational" glyph
(2) = "Phenotype prioritized" and use a person or patient or doctor looking glyph

This would help clarify and give some visual context for what the sources are.

**Comment by AlistairNWard** [comment]
Agreed, we need to think through the meaningful distinction of the sources and whether it's something someone who isn't us would even care about
